travis-ymls/marathon-client.travis.yml

```diff
@@ -0,0 +1,18 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : marathon-client
+# Source Repo         : https://github.com/willroden/marathon-client
+# Travis Job Link     : https://travis-ci.com/github/santosh653/marathon-client
+# Created travis.yml  : No
+# Maintainer          : Santosh Kulkarni <santoshkulkarni70@gmail.com>
+#
+# Script License      : Apache License, Version 2 or later
+#
+# ----------------------------------------------------------------------------
+# oraclejdk7/8/11 not supported on power hence running the code with latest openjdk11 on both Intel & Power to make it consistent.
```

Comment:
exclude oracle11 only on ppc64le.

Reply:
@Siddhesh-Ghadi Kindly verify now if the changes are correct? I have modified as per your comments.

Reply:
Please update the travis-job link with latest job. @santosh653

Reply:
@Siddhesh-Ghadi I have updated the Travis-job link as suggested by you. Kindly let me know if it is all right now.
Thanks !!